plugin-server/src/types.ts

```diff
@@ -756,13 +756,18 @@ export interface EventDefinitionType {
     created_at: string // DateTime
 }
 
+export type PropertyTypeFormat = 'unix_timestamp' | 'YYYY-MM-DD hh:mm:ss' | 'YYYY-MM-DD' // needs to sync with front end types
+export type PropertyType = 'DateTime' | 'String' | 'Numeric' | 'Boolean' // needs to sync with front end types
```

Comment:
If these are 100% synced, can we just use the Power of the Monorepo™ and import the frontend types directly? 🤔  It's an endless struggle to keep these types of things in sync.
Side note: is it feasible to make `PropertyType` an enum? `PropertyTypeFormat` may get pretty free-form, but looks like `PropertyType` is going to be a pretty strict set.

Reply:
+1 for enums, and for both types with the suggestion from above.

Reply:
The comments should really have been marked TODO. The types here are also introduced (ish) in #7892. And once that and this is merged we'd have introduced even more types where we're syncing between type files.

Looks like we'd need to use https://www.typescriptlang.org/docs/handbook/project-references.html. 

Better as a follow up PR?